schemas/app/PriceListItem.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "PriceListItem",
+  "label": "Price List Item",
+  "isChild": true,
+  "fields": [
+    {
+      "fieldname": "item",
+      "label": "Item",
+      "fieldtype": "Link",
+      "target": "Item",
+      "required": true,
+      "create": true,
+      "section": "Item"
+    },
+    {
+      "fieldname": "unit",
+      "label": "Unit Type",
+      "fieldtype": "Link",
+      "target": "UOM",
+      "section": "Item"
+    },
+    {
+      "fieldname": "rate",
+      "label": "Rate",
+      "fieldtype": "Currency",
+      "required": true,
+      "section": "Item"
+    }
+  ],
+  "quickEditFields": ["item", "unit", "rate"]
+}
```

Comment:
`PriceListItem` child table is invisible cause `tableFields` are not defined here.

![image](https://github.com/frappe/books/assets/60477442/9648dc7c-b659-48a1-b793-04a2a5ad12b7)

Reply:
Thanks! FB _really_ needs UI tests 🙈